### PR TITLE
Ansible playbook docs

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/install/ansible.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+parent: Install OpenSearch
+nav_order: 60
+---
+
+# Ansible playbook
+
+You can use an Ansible playbook to install and configure a production-ready OpenSearch cluster along with OpenSearch Dashboards.
+
+The Ansible playbook only supports deployment of OpenSearch and OpenSearch Dashboards to CentOS7 hosts.
+{: .note }
```
Comment (peterzhuamazon) [inline _opensearch/install/ansible.md:13]: Is this typo?

File: _opensearch/install/ansible.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+   ```bash
+   ansible-playbook -i inventories/opensearch/hosts opensearch.yml --extra-vars "admin_password=Test@123 kibanaserver_password=Test@6789"
+   ```
+
+   You can set the passwords for reserved users (`admin` and `kibanaserver`) using the `admin_password` and `kibanaserver_password` variables.
+
+2. After the deployment process is complete, you can access OpenSearch and OpenSearch Dashboards with the username `admin` and the password that you set for the `admin_password` variable:
+
+   ```bash
+   curl https://localhost:9200 -u 'admin:Test@123' --insecure
```
Comment (peterzhuamazon) [inline _opensearch/install/ansible.md:61]: Shouldnt default to localhost, need some clarification here I think.

If user bind IP to private IP or localhost, they have to be on the server that deployed this playbook to access.
If user bind IP to 0.0.0.0, then they use the public IP. They can also use private IP if they are in the same network.

File: _opensearch/install/ansible.md (hunk 1 above)
Comment (ashwinkumar12345) [reply _opensearch/install/ansible.md:13]: {: .note } makes the above sentence show up as a note.

**Review by peterzhuamazon — APPROVED** [review]
Thanks.

**Review by keithhc2 — APPROVED** [review]
LGTM

**Pull request merged**